src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/item.scss

```diff
@@ -44,3 +67,8 @@ $itemBorder: 1px $wildSand solid;
     vertical-align: middle;
     padding: 0 0 0 10px;
 }
+
+.dropdown-icon {
+    padding-left: 10px;
+    margin-right: -12px;
```

Comment:
From what I see in the design this statement is wrong (there should be 20px margin starting from the arrow).